### PR TITLE
Query features on map load

### PR DESCRIPTION
**Pull request by kuzkok**

### CODE REVIEW

File: addon/components/base-layer.js [hunk 1]
```diff
@@ -156,6 +156,28 @@ export default Ember.Component.extend(
...
+      @method _query
+      @param {Object} e Event object.
+      @param {Object} layerLinks Array contains layer links model, use for filter searched layers
+      @param {Object} queryFilter Object with query filter paramteres
+      @param {Object[]} results.features Array containing leaflet layers objects
+      or a promise returning such array.
+    */
+    _query(e) {
+      let layerId = this.get('layer.id').toString();
+      let layerLinks = e.layerLinks.filter(link => link.get('layer.id').toString() === layerId);
```
Comment (SeregaMatin) [inline addon/components/base-layer.js]: If we need layer links related only to current layer, why not to filter them in map-route where layer's hierarchy is loaded, & place them into each layer (related to them) as unstored property (like 'layers' for child layers in each layer model)?

Comment (kuzkok) [inline addon/components/base-layer.js]: query called only on map load, at this moment we have only exists layers.

File: addon/components/flexberry-map.js [hunk 2]
```diff
@@ -185,6 +185,24 @@ let FlexberryMapComponent = Ember.Component.extend(
...
+    layerLinks: null,
+
+    /**
+      JSON object with query parameters
+
+      @property queryFilter
+      @type object
+      @default null
+     */
+    queryFilter: null,
```
Comment (SeregaMatin) [inline addon/components/flexberry-map.js]: Maby it is better to pass queryFilter into 'flexberry-map:query' event handlers, and not store it as property of the layer instance.

File: addon/components/base-layer.js (hunk 1 above)
Comment (kuzkok) [inline addon/components/base-layer.js]: LayerLinks load by queryParameter, one map we can open with multiple settings each of one is may be fo different layer.

File: addon/components/flexberry-map.js [hunk 3]
```diff
@@ -271,6 +289,48 @@ let FlexberryMapComponent = Ember.Component.extend(
...
+
+              // Should alert user about not found objects
+              Ember.Logger.warn('Object not found for query', layerLinks, queryFilter);
+            }
+
+            // Hide map loader.
+            leafletMap.setLoaderContent('');
+            leafletMap.hideLoader();
+          });
+        });
```
Comment (SeregaMatin) [inline addon/components/flexberry-map.js]: Please, move logic of triggering & handling event into map-command.
And any possible errors aren't handled here, i see 'then', but i don't see 'catch'.

Comment (kuzkok) [inline addon/components/flexberry-map.js]: this is not command, it is load logic

Comment (kuzkok) [inline addon/components/flexberry-map.js]: yep, but at this time we cant call map command from map component at load :(, or can we do this?

Comment (SeregaMatin) [inline addon/components/flexberry-map.js]: Instantiate manually & call it's execute method.

**Pull request merged**